MCERemote/frmMCERemote.vb

```diff
@@ -1,19 +1,20 @@
+                                                {"Return", "28"}, {"Escape", "29"}, {"Backspace", "2A"}, {"Tab", "2B"}, {"Space", "2C"}, {"====================", ""}, {"OemMinus", "2D"}, {"OemPlus", "2E"}, {"OemOpenBracket", "2F"}, {"OemCloseBracket", "30"},
+                                                {"OemPipe", "31"}, {"OemSemiColon", "33"}, {"OemQuotes", "34"}, {"OemTilde", "35"}, {"OemComma", "36"}, {"OemPeriod", "37"}, {"OemQuestion", "38"},
+                                                {"CapsLock", "39"}, {"====================", ""}, {"F1", "3A"}, {"F2", "3B"}, {"F3", "3C"}, {"F4", "3D"}, {"F5", "3E"}, {"F6", "3F"}, {"F7", "40"}, {"F8", "41"}, {"F9", "42"}, {"F10", "43"}, {"F11", "44"}, {"F12", "45"}, {"====================", ""},
+                                                {"PrintScreen", "46"}, {"ScrollLock", "47"}, {"Break", "48"}, {"Insert", "49"}, {"Home", "4A"}, {"PageUp", "4B"}, {"PageDown", "4E"}, {"Delete", "4C"}, {"End", "4D"}, {"====================", ""}, {"Right", "4F"}, {"Left", "50"},
+                                                {"Down", "51"}, {"Up", "52"}, {"====================", ""}, {"NumLock", "53"}, {"NumPad /", "54"}, {"NumPad *", "55"}, {"NumPad -", "56"}, {"NumPad +", "57"}, {"NumPad Enter", "58"}, {"NumPad 1", "59"}, {"NumPad 2", "5A"}, {"NumPad 3", "5B"}, {"NumPad 4", "5C"},
+                                                {"NumPad 5", "5D"}, {"NumPad 6", "5E"}, {"NumPad 7", "5F"}, {"NumPad 8", "60"}, {"NumPad 9", "61"}, {"NumPad 0", "62"}, {"NumPad Del", "63"}, {"====================", ""}, {"App", "65"}, {"Keyboard Power", "66"}, {"Equals", "67"}, {"====================", ""}, {"F13", "68"}, {"F14", "69"}, {"F15", "6A"}, {"F16", "6B"},
+                                                {"F17", "6C"}, {"F18", "6D"}, {"F19", "6E"}, {"F20", "6F"}, {"F21", "70"}, {"D22", "71"}, {"F23", "72"}, {"F24", "73"}, {"====================", ""}, {"Execute", "74"}, {"Help", "75"}, {"Menu", "76"}, {"Select", "77"}, {"Stop", "78"}, {"Again", "79"}, {"Undo", "7A"}, {"Cut", "7B"}, {"Copy", "7C"},
+                                                {"Paste", "7D"}, {"Find", "7E"}, {"====================", ""}, {"Keyboard Mute", "7F"}, {"Keyboard VolumeUP", "80"}, {"Keyboard VolumeDown", "81"}, {"========== RAW INPUT MESSAGE", ""}, {"Shutdown", "8100"}, {"Sleep", "8200"}, {"Wake", "8300"}, {"========== WM_APPCOMMAND==", ""},
+                                                {"Mute", "E200"}, {"Info", "0902"}, {"VolumeUp", "E900"}, {"VolumeDown", "EA00"}, {"ChannelUp", "9C00"}, {"ChannelDown", "9D00"}, {"FastForward", "B300"}, {"Rewind", "B400"}, {"Play", "B000"}, {"Record", "B200"}, {"Pause", "B100"}, {"Stop", "B700"},
+                                                {"Next/Skip", "B500"}, {"Previous/Replay", "B600"}, {"TV Guide", "8D00"}, {"Play/Pause", "CD00"},

```

Comment:
Only difference is:
1. I added a new line for the browser keys.
2. I moved "Back" from before "TV Guide" to be on the list with browser keys.